handbook/company/pricing-features-table.yml

```
@@ -180,43 +448,84 @@
     - description: MDM commands for iOS/iPadOS are coming soon (2024-07-15).
     - description: Send MDM commands to tell end users to update their OS.
       moreInfoUrl: https://developer.apple.com/documentation/devicemanagement/schedule_an_os_update
+    - description: Configure agent options remotely, over the air.  (Includes osquery config, and osquery startup flags.).
+      moreInfoUrl: https://fleetdm.com/docs/configuration/agent-configuration
   productCategories: [Device management]
-  pricingTableCategories: [Device management]
-#
-#  ╔═╗╔═╗╦═╗╦╔═╗╔╦╗  ╔═╗═╗ ╦╔═╗╔═╗╦ ╦╔╦╗╦╔═╗╔╗╔
-#  ╚═╗║  ╠╦╝║╠═╝ ║   ║╣ ╔╩╦╝║╣ ║  ║ ║ ║ ║║ ║║║║
-#  ╚═╝╚═╝╩╚═╩╩   ╩   ╚═╝╩ ╚═╚═╝╚═╝╚═╝ ╩ ╩╚═╝╝╚╝
-- industryName: Script execution
-  friendlyName: Safely execute custom scripts (macOS, Windows, and Linux)
-  description: Deploy and execute custom scripts using a REST API, and manage your library of scripts in the UI or a git repo.
-  documentationUrl: https://fleetdm.com/docs/using-fleet/scripts
-  tier: Free
-  jamfProHasFeature: yes
-  jamfProtectHasFeature: no
-  dri: mikermcneil
+  pricingTableCategories: [Devices]
+#  
+#  ╔╦╗╔═╗╔═╗╦  ╔═╗╦═╗╔═╗╔╦╗╦╦  ╦╔═╗  ╔╦╗╔═╗╦  ╦╦╔═╗╔═╗  ╔╦╗╔═╗╔╗╔╔═╗╔═╗╔═╗╔╦╗╔═╗╔╗╔╔╦╗  
+#   ║║║╣ ║  ║  ╠═╣╠╦╝╠═╣ ║ ║╚╗╔╝║╣    ║║║╣ ╚╗╔╝║║  ║╣   ║║║╠═╣║║║╠═╣║ ╦║╣ ║║║║╣ ║║║ ║   
+#  ═╩╝╚═╝╚═╝╩═╝╩ ╩╩╚═╩ ╩ ╩ ╩ ╚╝ ╚═╝  ═╩╝╚═╝ ╚╝ ╩╚═╝╚═╝  ╩ ╩╩ ╩╝╚╝╩ ╩╚═╝╚═╝╩ ╩╚═╝╝╚╝ ╩            
+- industryName: Declarative Device Management (DDM) support for configuration profiles
+  description: Full support for Apple DDM configuration profiles.
+  documentationUrl: https://fleetdm.com/docs/using-fleet/mdm-os-updates#macos
+  tier: Premium
```

```suggestion
  tier: Free
```

Declaration (DDM) profiles are free.